docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+
+services:
+  redis:
+    image: redis:3.0.7-alpine
+    container_name: redis
+    command: redis-server --appendonly yes
+
+  compliance-checker:
+    image: test_ccweb:latest
+    container_name: compliance-checker-web
+    env_file: .env
```

Comment:
`.env` should already be the default environment file which is searched for.  You can leave this in if you wish, but this is just a heads-up.